connector.go

```diff
@@ -48,7 +48,8 @@ func (c *Connector) Deploy(ctx context.Context, image string) (deployer.Plugin,
 		SetVolumes(hostConfig.Binds).
 		SetCgroupNs(string(hostConfig.CgroupnsMode)).
 		SetNetworkMode(string(hostConfig.NetworkMode)).
-		SetPrivileged(hostConfig.Privileged)
+		SetPrivileged(hostConfig.Privileged).
+		SetReadOnlyRoot(&hostConfig.ReadonlyRootfs)
```

Comment:
This is why the dots belong at the beginning of the line rather than at the end:  when placed at the beginning, this change would have modified only one line instead of two....  😇

Reply:
Unfortunately, line endings are significant in Go, and so the dot _must_ go at the end of the line so that Go knows that the line continues.  😒